scripts/build_data.py

```
@@ -4441,6 +4460,14 @@ def strip_leading_numeric_locality_prefix(candidate: str) -> str:
     ).strip()
 
 
+def strip_leading_locality_preposition(candidate: str) -> str:
+    return re.sub(r"^(?:at|near)\s+", "", candidate, flags=re.IGNORECASE).strip()
+
+
+def strip_trailing_short_region_code(candidate: str) -> str:
+    return re.sub(r"\s+[A-Z]{2}$", "", candidate).strip(" -−ー－/()[]{}.")
```

Ruff (0.15.12)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)